src/lib.rs

```diff
@@ -14,7 +14,7 @@ use ipld_core::{
 };
 
 use crate::codec::PbNodeRef;
-pub use crate::{codec::PbNode, error::Error};
+pub use crate::{codec::PbLink, codec::PbNode, error::Error};
```

Comment:
As a rebase would be great I dare to leave a nitpick:

This could also be:

```suggestion
pub use crate::{codec::{PbLink, PbNode}, error::Error};
```

Reply:
Oh yeah! `rustfmt` just didn't do it for me, commiting